tests/unit/butterfree/migrations/test_cassandra_migration.py

```diff
@@ -0,0 +1,39 @@
+from butterfree.migrations import CassandraMigration
+
+
+class TestCassandraMigration:
+    def test_alter_table_query(self, dummy_db_schema, dummy_schema_diff):
+        cassandra_migration = CassandraMigration()
+
+        expected_query = "ALTER TABLE test ADD (kappa_column int, pogchamp uuid);"
+        query = cassandra_migration.create_query(
+            "test", dummy_db_schema, dummy_schema_diff
+        )
+
+        assert isinstance(query, str)
+        assert query, expected_query
+
+    def test_create_table_query(self, dummy_db_schema):
+
+        cassandra_migration = CassandraMigration()
+
+        expected_query = (
+            "CREATE TABLE test.test_table "
+            "(id int, platform text, ts bigint, PRIMARY KEY (id));"
+        )
+        query = cassandra_migration.create_query(
+            table_name="test_table", schema_diff=dummy_db_schema
+        )
+
+        assert isinstance(query, str)
+        assert query, expected_query
+
+    def test_no_diff(self, dummy_db_schema):
```

Comment:
`no_diff` should raise an error, because `create_query` shouldn't be called without a proper difference between database and feature set.

Reply:
Since ```schema_diff``` is now a required argument, I've removed this test. I can add, however, a test to check the arguments of ```create_query``` method. Do you think that this is necessary?